.github/workflows/wait_for_required_workflows.yml

```diff
@@ -135,7 +135,8 @@ jobs:
                 console.log("All required workflows have passed")
                 return
               }
-              
+              console.log(`Matching runs: ${matchingRuns}`)
```

Comment:
Maybe log only the names here instead of the whole objects?

Reply:
Good idea